hands-on-exercise/4-run-in-prod/1-run-prod-docker.md

```diff
@@ -1516,32 +1516,77 @@ Whenever you submit a transaction to `node-carol`, it will be propagated to the
 
 At this juncture, you may ask: Is it still possible to run a full game in almost a single block, as you did earlier in the [CosmJS integration tests](/hands-on-exercise/3-cosmjs-adv/2-cosmjs-messages.md#multiple-transactions-in-a-block)? After all, when `node-carol` passes on the transactions as they come, it is not certain that the recipients will honor the order in which they were received. Of course, they make sure to order Alice's transactions, thanks to the `sequence`, as well as Bob's. But do they keep the A-B-A-B... order in which they were sent?
 
-To find out:
+To find out, you need to credit tests' Alice and Bob's accounts:
```

Comment:
The first of the two possessives here is a little unclear to me. Is it to mean "You need to credit Alice and Bob's accounts in the test"?

Reply:
Ha. This run-prod thing has Alice and Bob and their sentries and validators.
The tests in `client` also have 2 accounts that are also named Alice and Bob. 
I want to prevent confusion. Here we want to put some coins on the Alice and Bob that are found in tests.